drydock/templates/kustomized/global-stack/kustomization.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I created a global-stack for manifests that don't depend on the tutor installation but I'm not quite sure how we could manage multiple resources

Reply:
For the time being we can have the resources in this single kustomization file. We can add a patch to add additional resources and helmCharts.

```yaml
resources:
  patch("drydock-global-resources")

helmCharts:
  ...
  patch("drydock-global-helm-charts")
```

Reply:
Should we use just patches instead of hardcoding the prometheus chart?